sources/packages/web/src/services/http/common/HttpBaseClient.ts

```diff
@@ -13,8 +14,14 @@ export default abstract class HttpBaseClient {
       throw new Error("User is not authenticated!");
     }
   }
+  static renewTokenIfExpired() {
+    if (AuthService.shared.keycloak?.isTokenExpired(MINIMUM_TOKEN_VALIDITY)) {
+      AuthService.shared.keycloak?.updateToken(MINIMUM_TOKEN_VALIDITY);

```

Comment:
This method returns a promise. All chain off calls must be changed to await, right? Am I missing something?